test/engine/test_parseconnect.py

```diff
@@ -196,6 +198,18 @@ def test_comparison(self):
         is_true(url1 != url3)
         is_false(url1 == url3)
 
+    def test_warnings(self):
+        assert_raises(
+            exc.SADeprecationWarning,
+            url.URL,
+            "dbtype"
+        )
+        url1 = url.URL.create("dbtype")
+        url2 = copy.copy(url1)
+        url3 = copy.deepcopy(url2)
+        is_true(url1 == url2)
```

Comment:
we have a fixture `eq_()` for this, `eq_(url1, url2)`.   it's not any better or worse we just try to be consistent.